### PR TITLE
Private - Events App

### PR DESCRIPTION
**Pull request by acushlakoncept**
This Pull Request contains the complete private events app that allows users to create events and then manage user signups. Users can create events.
We gained a lot of insights on rails associations and scopes.

### CODE REVIEW

File: db/migrate/20200806184736_create_invitations.rb [hunk 1]
```diff
@@ -0,0 +1,10 @@
+class CreateInvitations < ActiveRecord::Migration[6.0]
+  def change
+    create_table :invitations do |t|
+      t.integer :attendee_id
+      t.integer :attended_event_id
+
+      t.timestamps
+    end
```
Comment (maelfosso) [inline db/migrate/20200806184736_create_invitations.rb:8]: Kindly add indexes here

File: app/controllers/sessions_controller.rb [hunk 2]
```diff
@@ -0,0 +1,23 @@
+class SessionsController < ApplicationController
+  def new; end
+
+  def create
+    @user = User.find_by(name: params[:session][:user])
+    # @user ||= User.find(params[:session][:user].to_i)
```
Comment (maelfosso) [inline app/controllers/sessions_controller.rb:6]: Remove comments

File: app/controllers/events_controller.rb [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+    @event = Event.find(params[:id])
+  end
+
+  # def self.past
+  #     Event.where(["date < ?", Date.today])
+  # end
+
+  # def self.upcoming
+  #     Event.where(["date > ?", Date.today])
+  # end
```
Comment (maelfosso) [inline app/controllers/events_controller.rb:35]: Remove comments, please

File: db/migrate/20200806184736_create_invitations.rb (hunk 1 above)
Comment (acushlakoncept) [reply db/migrate/20200806184736_create_invitations.rb:8]: Added the indexes

File: app/controllers/events_controller.rb (hunk 3 above)
Comment (acushlakoncept) [reply app/controllers/events_controller.rb]: Thank you, we have added unit test and integration test and it's running perfectly well now.

**Pull request merged**